Bonsai.NuGet.Design/PackageViewController.cs

```diff
@@ -227,68 +217,9 @@ static Bitmap ResizeImage(Image image, Size newSize)
             return result;
```

Comment:
`ResizeImage` was moved to `IconReader` and should be removed from `PackageViewController` as it is no longer referenced.

Reply:
Good catch!